src/pages/permissions/PermissionIdpGroups.tsx

```diff
@@ -0,0 +1,327 @@
+          className: "u-align--right",
+          "aria-label": "Number of mapped groups",
+        },
+        {
+          className: "actions u-align--right",
+          content: (
+            <List
+              inline
+              className="u-no-margin--bottom actions-list"
+              items={[
+                <Button
+                  key={`edit-${idpGroup.name}`}
+                  appearance="base"
+                  hasIcon
+                  dense
+                  onClick={() => panelParams.openEditIdpGroup(idpGroup.name)}
+                  type="button"
+                  aria-label="Edit IDP group details"
+                  title="Edit details"
+                >
+                  <Icon name="edit" />
+                </Button>,
+                <DeleteIdepGroupBtn
+                  key={`delete-${idpGroup.name}`}
+                  idpGroup={idpGroup}
+                />,
+              ]}
+            />
+          ),
+          role: "cell",
+          "aria-label": "Actions",
+        },
+      ],
+      sortData: {
+        name: idpGroup.name.toLowerCase(),
+        groups: idpGroup.groups.length,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  if (isLoading) {
+    return <Loader text="Loading identity provider groups" />;
+  }
+
+  const getTablePaginationDescription = () => {
+    if (selectedGroupNames.length > 0) {
+      return (
+        <SelectedTableNotification
+          totalCount={groups.length ?? 0}
+          itemName="IDP group"
+          parentName=""
+          selectedNames={selectedGroupNames}
+          setSelectedNames={setSelectedGroupNames}
+          filteredNames={filteredGroups.map((item) => item.name)}
+          hideActions={!!panelParams.panel}
+        />
+      );
+    }
+  };
+
+  const hasGroups = groups.length > 0;
+  const content = hasGroups ? (
+    <>
+      <Notification severity="information" title="IDP group mappings">
+        <>
+          Identity provider groups map authentication entities from your
+          identity provider to groups within LXD.{" "}
+          {!hasCustomClaim ? (
+            <>
+              <span>You need to set your server</span>{" "}
+              <Link to="/ui/settings">
+                configuration (<code>oidc.groups.claim</code>)
+              </Link>{" "}
+              <span>
+                to the name of the custom claim that provides the IDP groups.
+              </span>
+            </>
+          ) : (
+            ""
+          )}
+          <p className="u-no-margin--bottom">
+            <a
+              href={`${docBaseLink}/explanation/authorization/#use-groups-defined-by-the-identity-provider`}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Learn more about IDP groups
+              <Icon className="external-link-icon" name="external-link" />
+            </a>
+          </p>
+        </>
```

Comment:
```suggestion        <>
          <p>
            Identity provider groups map authentication entities from your
            identity provider to groups within LXD.
          </p>
          {!hasCustomClaim ? (
            <p>
              You need to set your server{" "}
              <Link to="/ui/settings">
                configuration (<code>oidc.groups.claim</code>)
              </Link>{" "}
              to the name of the custom claim that provides the IDP groups.
            </p>
          ) : (
            ""
          )}
          <a
            href={`${docBaseLink}/explanation/authorization/#use-groups-defined-by-the-identity-provider`}
            target="_blank"
            rel="noopener noreferrer"
          >
            Learn more about IDP groups
            <Icon className="external-link-icon" name="external-link" />
          </a>
        </>
```
I'd suggest this for a more straight forward usage of HTML tags and to improve spacing.

We should follow the DRY pattern. Better reuse, maybe extract into a dedicated component instead of repeating this same code below for the empty state.

Reply:
I think a separate component might be an overkill, this text is unlikely to be used anywhere else. Assigning to a variable seems appropriate. Also we should put a `<p>` tag around the link so that the button in empty state does not appear inline next to the link.